.github/ISSUE_TEMPLATE/documentation_report.md

```diff
@@ -2,37 +2,35 @@
 name: 📝 Documentation Report
 about: Ask us about docs
 ---
+## Documentation report
 <!--- Verify first that your improvement is not already reported on GitHub -->
 <!--- Also test if the latest release and devel branch are affected too -->
 <!--- Complete *all* sections as described, this form is processed automatically -->
 
-##### SUMMARY
+### Summary
 <!--- Explain the problem briefly below, add suggestions to wording or structure -->
 
 <!--- HINT: Did you know the documentation has an "Edit on GitHub" link on every page ? -->
 
-##### ISSUE TYPE
-- Documentation Report
```

Comment:
Did this mean to be removed?

Reply:
Yes, it's now in the title. Makes more sense as it is pre-filled.

@webknjaz argued that people may remove it, but if it is no longer a section, there's less chance people will be editing.